SpatialGDK/Source/SpatialGDK/Private/Interop/RPCs/MulticastRPCService.cpp

```diff
@@ -116,6 +131,25 @@ void MulticastRPCService::EntityAdded(const Worker_EntityId EntityId)
 	}
 }
 
+void MulticastRPCService::EntityRefresh(Worker_EntityId EntityId)
+{
+	for (const Worker_ComponentSetId ComponentSetId : SubView->GetView()[EntityId].Authority)
+	{
+		if (ComponentSetId == SpatialConstants::SERVER_AUTH_COMPONENT_SET_ID)
```

Comment:
Can we assume this from completeness? Should a completeness refresh mean we don't have auth at this point?

Reply:
IIRC, the multicast view is wired on the actors subview, so you have to check auth manually.